src/components/TypingIndicator/TypingIndicator.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+
+import { AvatarProps, Avatar as DefaultAvatar } from '../Avatar';
+
+import { useChannelContext } from '../../context/ChannelContext';
+
+import type {
+  DefaultAttachmentType,
+  DefaultChannelType,
+  DefaultCommandType,
+  DefaultEventType,
+  DefaultMessageType,
+  DefaultReactionType,
+  DefaultUserType,
+  UnknownType,
+} from '../../../types/types';
+
+export type TypingIndicatorProps = {
+  /**
+   * Custom UI component to display user avatar.
+   *
```

Comment:
delete this space, been trying to do that everywhere else to tighten things up a bit